libraries/chain/include/eosio/chain/config.hpp

```diff
@@ -48,6 +48,7 @@ const static uint64_t domain_account_name    = N(cyber.domain);
 const static uint64_t history_account_name   = N(.history);
 const static uint64_t govern_account_name    = N(cyber.govern);
 const static uint64_t stake_account_name     = N(cyber.stake);
+const static uint64_t snapshot_writer_name  = N(.snapshot);
```

Comment:
For what it is used? I haven't find.

Reply:
It used in libraries/chain/include/cyberway/chaindb/noscope_tables.hpp to switch off filtering by scope

Reply:
Yep, I saw it.
But `.snapshot` doesn't have tables, that is why I don't understand for what it is .

Reply:
I think .snapshot is not a good name because api method get_tables_by_scope also needs to go through all scopes. I think about more universal name